### PR TITLE
docs: add remote faq for older linux versions

### PR DESCRIPTION
**Pull request by deepak1556**
Needed for https://github.com/microsoft/vscode/pull/202210

### CODE REVIEW

File: docs/remote/faq.md [hunk 1]
```diff
@@ -101,12 +101,21 @@ You can use one of the following solutions to resolve this problem:
...
+Remote Development requires kernel >= 4.18, glibc >=2.28, and libstdc++ >= 3.4.25. Recent x86_64 glibc-based distributions have the best support, but exact requirements can vary by distribution.
 
 Support for musl-based [Alpine Linux](https://alpinelinux.org) is available for the Dev Containers and WSL extensions and ARMv7l (AArch32) / ARMv8l (AArch64) is available in Remote - SSH. However, native dependencies in certain extensions may cause them not to function on non-x86_64 glibc distributions. Note that experimental ARMv8l (AArch64) is available in [VS Code Insiders](https://code.visualstudio.com/insiders/) only.
 
 See [Remote Development with Linux](/docs/remote/linux.md) for additional details.
 
+### Can I run VS Code Server on older Linux distributions?
+
+Starting with `1.86` release of VS Code, the minimum toolchain for building the remote server was raised after assessment and the prebuilt servers are compatible with Linux distributions based on glibc 2.28 or later, for example, Debian 10, RHEL 8, Ubuntu 20.04.
```
Comment (joaomoreno) [inline docs/remote/faq.md:112]:
```suggestion
Starting with `1.86` release of VS Code, the minimum requirements for the build toolchain of the remote server were raised. The prebuilt servers distributed by VS Code are compatible with Linux distributions based on glibc 2.28 or later, for example, Debian 10, RHEL 8, Ubuntu 20.04.
```

File: docs/remote/faq.md [hunk 2]
```diff
@@ -101,12 +101,21 @@ You can use one of the following solutions to resolve this problem:
...
+
+If your platform does not meet the requirements and you are unable to upgrade the Linux distribution, you can downgrade the VS Code client to version `1.85` to continue usage of the Remote Development. Following are the ways to dowgrade the VS Code client on desktop and web,
```
Comment (joaomoreno) [inline docs/remote/faq.md:114]:
```suggestion
If your platform does not meet the requirements and you are unable to upgrade the Linux distribution, you can downgrade the VS Code client to version `1.85` to continue usage of the Remote Development. You can downgrade the VS Code client both on desktop and web:
```

File: docs/remote/faq.md [hunk 3]
```diff
@@ -101,12 +101,21 @@ You can use one of the following solutions to resolve this problem:
...
+
+* For desktop, you can download `1.85` release from [here](https://code.visualstudio.com/updates/v1_85). Depending on your platform ensure to pin the package version in apt or exclude it in yum to prevent automatic updates.
```
Comment (joaomoreno) [inline docs/remote/faq.md:116]:
```suggestion
* On desktop, you can download the `1.85` release from [here](https://code.visualstudio.com/updates/v1_85). Depending on your platform, make sure to disable updates to stay on that version. A good recommendation is to have 1.85 as a separate installation, set up with [Portable Mode](https://code.visualstudio.com/docs/editor/portable). That way, you won't affect your main desktop VS Code version.
```

File: docs/remote/faq.md [hunk 4]
```diff
@@ -101,12 +101,21 @@ You can use one of the following solutions to resolve this problem:
...
+* For web, you can add the following query argument [?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2](https://vscode.dev/?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2) to force `1.85` version of the client.
```
Comment (joaomoreno) [inline docs/remote/faq.md:117]:
```suggestion
* On web, you can add the following query argument [?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2](https://vscode.dev/?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2) to use `1.85`.
```

File: docs/remote/faq.md [hunk 5]
```diff
@@ -101,12 +101,21 @@ You can use one of the following solutions to resolve this problem:
...
+Remote Development requires kernel >= 4.18, glibc >=2.28, and libstdc++ >= 3.4.25. Recent x86_64 glibc-based distributions have the best support, but exact requirements can vary by distribution.
 
 Support for musl-based [Alpine Linux](https://alpinelinux.org) is available for the Dev Containers and WSL extensions and ARMv7l (AArch32) / ARMv8l (AArch64) is available in Remote - SSH. However, native dependencies in certain extensions may cause them not to function on non-x86_64 glibc distributions. Note that experimental ARMv8l (AArch64) is available in [VS Code Insiders](https://code.visualstudio.com/insiders/) only.
 
 See [Remote Development with Linux](/docs/remote/linux.md) for additional details.
 
+### Can I run VS Code Server on older Linux distributions?
+
+Starting with `1.86` release of VS Code, the minimum requirements for the build toolchain of the remote server were raised. The prebuilt servers distributed by VS Code are compatible with Linux distributions based on glibc 2.28 or later, for example, Debian 10, RHEL 8, Ubuntu 20.04.
```
Comment (ntrogh) [inline docs/remote/faq.md:112]:
```suggestion
Starting with VS Code release 1.86, the minimum requirements for the build toolchain of the remote server were raised. The prebuilt servers distributed by VS Code are compatible with Linux distributions based on glibc 2.28 or later, for example, Debian 10, RHEL 8, Ubuntu 20.04.
```

File: docs/remote/faq.md [hunk 6]
```diff
@@ -101,12 +101,21 @@ You can use one of the following solutions to resolve this problem:
...
+
+If your platform does not meet the requirements and you are unable to upgrade the Linux distribution, you can downgrade the VS Code client to version `1.85` to continue usage of the Remote Development. You can downgrade the VS Code client both on desktop and web:
```
Comment (ntrogh) [inline docs/remote/faq.md:114]:
```suggestion
If your setup does not meet these requirements and you are unable to upgrade the Linux distribution, you can downgrade the VS Code client to version 1.85 to continue using Remote Development. You can downgrade the VS Code client on both desktop and web:
```

File: docs/remote/faq.md [hunk 7]
```diff
@@ -101,12 +101,21 @@ You can use one of the following solutions to resolve this problem:
...
+
+* On desktop, you can download the `1.85` release from [here](https://code.visualstudio.com/updates/v1_85). Depending on your platform, make sure to disable updates to stay on that version. A good recommendation is to have 1.85 as a separate installation, set up with [Portable Mode](https://code.visualstudio.com/docs/editor/portable). That way, you won't affect your main desktop VS Code version.
```
Comment (ntrogh) [inline docs/remote/faq.md:116]:
```suggestion
* On desktop, you can download the VS Code release 1.85 from [here](https://code.visualstudio.com/updates/v1_85). Depending on your platform, make sure to disable updates to stay on that version. A good recommendation is to have release 1.85 as a separate installation, set up with [Portable Mode](https://code.visualstudio.com/docs/editor/portable). That way, you won't affect your main desktop VS Code version.
```

File: docs/remote/faq.md [hunk 8]
```diff
@@ -101,12 +101,21 @@ You can use one of the following solutions to resolve this problem:
...
+* On web, you can add the following query argument [?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2](https://vscode.dev/?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2) to use `1.85`.
```
Comment (ntrogh) [inline docs/remote/faq.md:117]:
```suggestion
* On web, you can add the following query argument [`?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2`](https://vscode.dev/?vscode-version=0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2) to use VS Code release 1.85.
```

**Review by ntrogh — CHANGES_REQUESTED** [review]
Minor text updates

**Review by ntrogh — APPROVED** [review]

**Pull request merged**